utils/message-generator/src/parser/actions.rs

```diff
@@ -4,16 +4,22 @@ use roles_logic_sv2::parsers::AnyMessage;
 use serde_json::{Map, Value};
 use std::collections::HashMap;
 
+/// Represents actions to parse.
+// TODO: should struct have an `actions` field of type `Vec<Action<'a>>`?
```

Comment:
@Fi3 do we want to change the `ActionParser` struct to more closely match how `Frames` is where its `from_step_1` return is a `Frames` struct field? So like:

```
pub struct ActionParser {
  pub actions: Vec<Action<'a>>
}
```

Reply:
Yes, update `Actions` to match `Frames` formatting.
Also can move `Step3` match statement logic in `parsers/mod.rs` to a command.rs file for consistency